MANUAL.md

```diff
@@ -41,13 +41,19 @@ You can install and run this wizard like this:
 
     `rmt-cli systems remove SCC_e740f34145b84523a184ace764d0d597`
 
-  * `rmt-cli products list [--all] [--csv]`:
+  * `rmt-cli products list [--all] [--csv] [--name name] [--version version] [--arch arch]`:
     Lists the products that are enabled for mirroring.
 
     Use the `--all` flag to list all available products.
 
     Use the `--csv` flag to output the list in CSV format.
 
+	Use the `--name name` flag to list all products whose name match `name`.
+
+	Use the `--version ver` flag to list all products whose version match `version`.
+
+	Use the `--arch arch` flag to list all products whose architecture  match `arch`.
```

Comment:
These lines are indented differently than the rest of this file. Maybe you've used a tab instead of two spaces? Can you please align the formatting?